platform/ios/src/MGLCalloutView.h

```diff
@@ -8,32 +8,38 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol MGLAnnotation;
 
 /**
- A protocol for a `UIView` subclass that displays information about a selected annotation near that annotation.
+ A protocol for a `UIView` subclass that displays information about a selected
+ annotation near that annotation.
  */
 @protocol MGLCalloutView <NSObject>
 
 /**
- An object conforming to the `MGLAnnotation` protocol whose details this callout view displays.
+ An object conforming to the `MGLAnnotation` protocol whose details this callout
+ view displays.
  */
 @property (nonatomic, strong) id <MGLAnnotation> representedObject;
 
 /**
- A view that the user may tap to perform an action. This view is conventionally positioned on the left side of the callout view.
+ A view that the user may tap to perform an action. This view is conventionally
+ positioned on the left side of the callout view.
  */
 @property (nonatomic, strong) UIView *leftAccessoryView;
 
 /**
- A view that the user may tap to perform an action. This view is conventionally positioned on the right side of the callout view.
+ A view that the user may tap to perform an action. This view is conventionally
+ positioned on the right side of the callout view.
  */
 @property (nonatomic, strong) UIView *rightAccessoryView;
 
 /**
- An object conforming to the `MGLCalloutViewDelegate` method that receives messages related to the callout view’s interactive subviews.
+ An object conforming to the `MGLCalloutViewDelegate` method that receives
+ messages related to the callout view’s interactive subviews.
  */
 @property (nonatomic, weak) id<MGLCalloutViewDelegate> delegate;
 
 /**
- Presents a callout view by adding it to `inView` and pointing at the given rect of `inView`’s bounds. Constrains the callout to the bounds of the given view.
+ Presents a callout view by adding it to `inView` and pointing at the given rect
```

Comment:
While you’re at it, can you rename `inView` to just `view`? `inView:` is part of the selector; `view` is the name of the parameter.